File: app/src/main/java/com/yarashevich/kiryl/ipd/ExampleMaterialAboutFragment.java
package com.yarashevich.kiryl.ipd;

import android.content.Context;

import com.example.library.MaterialAboutFragment;
import com.example.library.model.MaterialAboutList;


public class ExampleMaterialAboutFragment extends MaterialAboutFragment {

    @Override
    protected MaterialAboutList getMaterialAboutList(final Context c) {
        return Demo.createMaterialAboutList(c, R.color.colorIconLight, THEME_LIGHT);
    }

    @Override
    protected int getTheme() {
        return THEME_LIGHT;
    }
}

File: app/src/main/java/com/yarashevich/kiryl/ipd/ExampleMaterialAboutFragment.kt
package com.yarashevich.kiryl.ipd

import android.content.Context

import com.example.library.MaterialAboutFragment
import com.example.library.model.MaterialAboutList


class ExampleMaterialAboutFragment : MaterialAboutFragment() {

    override fun getMaterialAboutList(c: Context): MaterialAboutList {
        return Demo.createMaterialAboutList(c, R.color.colorIconLight, MaterialAboutFragment.THEME_LIGHT)
    }

    override fun getTheme(): Int {
        return MaterialAboutFragment.THEME_LIGHT
    }
}
